org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Word/ImportNarrativeDocument.java

```diff
@@ -699,6 +699,32 @@ public NarrEntry(final String entry) throws ParseException
   {
     boolean askYes(String title, String message);
   }
+  
+  public static interface TrimNarrativeHelper
+  {
+    ImportNarrativeEnum findWhatToImport();
+  }
+  
+  public static enum ImportNarrativeEnum{
+    TRIMMED_DATA("trimmed-data"), 
+    ALL_DATA("all-data"),
+    CANCEL("cancel");
+    private String name;
+    ImportNarrativeEnum(String string){
+      this.name = string;
+    }
+    public String getName() {
+      return this.name;
+    }
+    public static ImportNarrativeEnum getByName(String name) {
+      switch(name) {
+        case "trimmed-data":return TRIMMED_DATA;
```

Comment:
@ayeshare - you're going beyond my ability with enums, here - so I'm keen to learn.   But, wouldn't it be better to replace these duplicated strings (`"trimmed-data"`) with final variables?

Reply:
I can replace it with a constant but in enum the definition of Enum should be the very first statement.  So it will still be duplicated.

Reply:
Ok, workaround applied:
![image](https://user-images.githubusercontent.com/1108513/44862770-51e55280-ac73-11e8-9cce-bba51b42992c.png)